lib/gollum/templates/layout.mustache

```diff
@@ -90,7 +102,23 @@
   </script>
   {{/mermaid}}
   
-  {{#js}}<script type="text/javascript" src="{{custom_js}}"></script>{{/js}}
+  {{#js}}
+  <script type="text/javascript" src="{{custom_js}}"></script>
+  {{/js}}
+  {{#math_config}}
+  <script type="text/javascript" src="{{math_config}}"></script>
+  {{/math_config}}
+
+  {{#mathjax}}
+  <script async src="{{mathjax_js}}"></script>
+  {{/mathjax}}
+  {{#katex}}
+  <script type="text/javascript">
+    document.addEventListener("DOMContentLoaded", function() {
+        renderMathInElement(document.body, katex_conf);
```

Comment:
We could also try passing in the wiki content div instead of `document.body`.

Reply:
Makes sense to me, as all markup language (and katex blocks) should be contained in `wiki-content`.